Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -167,6 +167,9 @@ struct SeedFinderConfig {
   // Returns position of the center of the top strip.
   Delegate<Acts::Vector3(const SpacePoint&)> getTopStripCenterPosition;
 
+  // Delegate to apply experiment specific cuts
+  Delegate<bool(const float bottomRadius, const float cotTheta)> experimentCuts;
```

Comment:
Is t he `const` really needed? From the interface point of view, it doesn't really matter, since it's a copy anyway.

Reply:
This could potentially be defaulted to the "always `true`" case you set in the two seeding algorithms below. Something like this would work I think:

```suggestion
  Delegate<bool(float bottomRadius, float cotTheta)> experimentCuts = [](){
  	Delegate<bool(float bottomRadius, float cotTheta)> delegate;
  	delegate.connect<&noopExperimentCuts>();
  	return delegate;
  }();
```

with
```cpp
inline bool noopExperimentCuts(float /*bottomRadius*/, float /*cotTheta*/) { 
   return true; 
}
```
somewhere above.

Reply:
I don't fully understand the reason for the doxygen failure `Invalid C++ declaration: Expected identifier in nested name`